### PR TITLE
For the purposes of negative globs, treat them as positive globs.

### PR DESCRIPTION
**Pull request by krainboltgreene**
In Github you can define a "negative glob" aka, "not this path". To still validate this type of glob we simply check the opposite of that path.

### CODE REVIEW

**Review by jstuessy — APPROVED** [review]

**Comment by mpalmer** [comment]
Hi Kurtis, good catch.  I've never used a negative glob myself, so I guess that's why they don't work.  :joy:

Clippy's got a bone to pick with you -- although I would have expected a regular build to fail, too, so I'll ponder what's going on there some other time.  If you can add a couple of tests to the pile in `test` (maybe `003a_successful_negative_globs` and `004a_failing_negative_globs`?) that'll let me just click the "Merge" button when all the checks are green.

**Comment by krainboltgreene** [comment]
Yeah I'm not sure what the problems here are, as all the documentation details this function as taking 2 arguments.

**Comment by krainboltgreene** [comment]
oh, lol there's a replace function on `Option`: https://doc.rust-lang.org/std/option/enum.Option.html#method.replace

**Comment by mpalmer** [comment]
Thanks for the updates, Kurtis.  I'll get onto this by next week, I promise. :crossed_fingers:

**Comment by krainboltgreene** [comment]
@mpalmer Any chance you could run the actions?

**Comment by mpalmer** [comment]
Sorry, I thought once I'd approved them once they'd be run automatically thereafter.  TIL...

**Comment by mpalmer** [comment]
I created a second PR, #71, based on yours, and fixed up a few things.  It's now merged, so we can consider this done, I reckon.  Thanks for your work to make it happen!

**Pull request merged**
